### PR TITLE
[tensorboard] fix add_histogram_raw

### PR DESCRIPTION
**Pull request by lanpa**
This is a porting of the fix from:
https://github.com/lanpa/tensorboardX/issues/421

cc @orionr

### CODE REVIEW

File: test/test_tensorboard.py [hunk 1]
```diff
@@ -105,7 +105,7 @@ def test_pytorch_histogram_raw(self):
                                     num=num,
                                     sum=floats.sum().item(),
                                     sum_squares=sum_sq,
-                                    bucket_limits=limits.tolist(),
+                                    bucket_limits=limits[1:].tolist(),
```
Comment (orionr) [inline test/test_tensorboard.py]: Nice!

**Review by orionr — APPROVED** [review]
Thank you!

**Review by orionr — COMMENTED** [review]
Actually, @lanpa, can you bring the example over from tensorboardX here as well? I think that will be useful. Thanks!

File: torch/utils/tensorboard/writer.py [hunk 2]
```diff
@@ -385,13 +385,48 @@ def add_histogram_raw(self, tag, min, max, num, sum, sum_squares,
...
+            counts, limits = np.histogram(values, bins=bins)
+            sum_sq = values.dot(values)
+            writer.add_histogram_raw(
+                tag='histogram_with_raw_data',
+                min=values.min(),
+                max=values.max(),
+                num=len(values),
+                sum=values.sum(),
+                sum_squares=sum_sq,
+                bucket_limits=limits.tolist(),
```
Comment (orionr) [inline torch/utils/tensorboard/writer.py]: Shouldn't this be

```
bucket_limits=limits[1:].tolist(),
```
?

Comment (lanpa) [reply torch/utils/tensorboard/writer.py]: Ahrr~ you are right.

**Pull request closed without merging**